content/BPIP-5.md

```diff
@@ -151,47 +152,86 @@ The following methods are added.
 
 ```solidity
     /**
-     * @notice Gets EIP2981 style royalty information for a chosen exchange.
+     * @notice Gets EIP2981 style royalty information for a chosen offer or exchange.
      *
-     * EIP2981 supports only 1 recipient, there fore this method defaults to treasury address.
+     * EIP2981 supports only 1 recipient, therefore this method defaults to treasury address.
      * This method is not exactly compliant with EIP2981, since it does not accept `salePrice` and does not return `royaltyAmount,
-     * but it rather returns `royaltyPercentage` which is the sum of all bps (exchange can have multiple royalty recipients).
+     * but it rather returns `royaltyPercentage` which is the sum of all bps (an exchange can have multiple royalty recipients).
      *
-     * This function is meant to be primarily used by boson voucher client, which implements EIP2981.
+     * This function is meant to be primarly used by boson voucher client, which implements EIP2981.
```

Comment:
Eh, wasn't the original correct? :)

Reply:
Yes, it was. 
I copied the text from the contracts repo, where this typo somehow sneaked in.

It's fixed now.